prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -1421,6 +1421,8 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
     ) {
         val ctx = if (txn != null) DSL.using(txn) else create
         ctx.update(PERMISSION)
+            .set(PERMISSION.NAME, permission.name)
```

Comment:
Does this change mean the client HAS to provide the name and description otherwise it will get set to empty? If so, I think instead of permission.name we should check for null or empty and if so write the current value.

Reply:
I was thinking the UI would pass the name, description, and enabled values always. This means some/all could be modified, otherwise it would be passing the original values. Thoughts?

Reply:
Sure, if the front-end took it upon itself to do that then we don't have any problems, in that particular case. But from the perspective of the API it would be better if it checked for empty/null before setting it, since the API can't control what the client sends and the API knows that empty/null values are not valid. For example, what if someone calls the API via postman or a third party application and forgets to fill out all the values?

Now that I think about it more, we would probably have to do a DB read to get the current value, right? So instead of `we should check for null or empty and if so write the current value.` I should have said `we should check for null or empty and if so DON'T set that value in the query syntax. ` This probably means you would have to do the .set calls on separate lines as part of a conditional.

As a side note, if we did query params we could probably mark the params as required and the azure function would reject the request automatically, saving us from some work. But, in my opinion, name and description should be optional anyways.

Reply:
Also, I can see this being part of a different ticket, so feel free to create one and I can approve this PR, since it is just for documentation.

Reply:
@arnejduranovic makes sense. Thanks!